### DISCUSSION TITLE
{f1dataR} New R package (on CRAN!) wrapping FastF1

### DISCUSSION THREAD

**Discussion by pbulsink** [theOehrly/Fast-F1#437] [Show and tell]
Hello

Just to let you know, we recently released on CRAN an R package that makes FastF1 data available in R. 

For anyone looking to use this package's core components, check out {f1dataR}. 

I'm not sure if you want to share this on the Examples page or not, but we felt you should know.

Thanks for all your work!

**Comment by theOehrly** [comment]
That's really cool. I'll take a look at it and probably add a note to the installation section as a recommendation for people who prefer to work with the data in R. I'll get back to you here with any questions, in case I have any.

**Comment by theOehrly** [comment]
@pbulsink I finally added a link to the R package in the Readme and main page of the docs :)
